build/haumea/zrepl.nix

```diff
@@ -33,23 +33,27 @@
       };
       pruning = {
         keep_sender = [
+          { type = "not_replicated"; }
```

Comment:
Oops, I expect we need have the `regex =` part here as well.

Reply:
Or maybe not.  Their examples set doesn't have it:
https://zrepl.github.io/configuration/prune.html#pruning-policies

Reply:
Maybe we should drop this line anyway.  In case the remote end is down, we probably want to keep pruning the sender to reduce the risk of running out of space.